lms/config/__init__.py

```diff
@@ -14,11 +14,11 @@ def configure(settings):
     env_settings = {
         # The URL of the https://github.com/hypothesis/via instance to
         # integrate with.
-        "via_url": sg.get("VIA_URL", required=True),
-        "jwt_secret": sg.get("JWT_SECRET", required=True),
-        "google_client_id": sg.get("GOOGLE_CLIENT_ID", required=True),
-        "google_developer_key": sg.get("GOOGLE_DEVELOPER_KEY", required=True),
-        "google_app_id": sg.get("GOOGLE_APP_ID", required=True),
+        "via_url": sg.get("VIA_URL"),
+        "jwt_secret": sg.get("JWT_SECRET"),
+        "google_client_id": sg.get("GOOGLE_CLIENT_ID"),
+        "google_developer_key": sg.get("GOOGLE_DEVELOPER_KEY"),
+        "google_app_id": sg.get("GOOGLE_APP_ID"),
```

Comment:
The `required=True` makes it impossible to run the Pyramid app at all (e.g. to run a command line script to set those envvars) without those envvars already being set. I've just removed the concept of a required envvar that crashes on startup unless set. It's unnecessary and has only gotten in the way of people setting up their envs manually, and now gets in the way of scripting

Reply:
I'm just removing it because it's more trouble than it's worth. With the new `development.ini` defaults on this branch, plus `make devdata`, it's indeed not needed in dev if you have access to devdata. I don't think this app is of much interest outside of our team, so not so worried about people who don't have devdata (they will now be able to launch the app with required settings not set, and it will crash when it tries to use them). In a QA or prod deploy it will be possible to launch the app with required settings missing and it'll crash when it tries to use them, but only two QA/prod deploys exist and both have all the settings already. In both of those cases we do have manual setup docs that tell you to set up all these settings (those docs do seem unlikely to be maintained, though). So it's just not providing much benefit and it gets in the way.